runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/WriteFilesTranslation.java

```diff
@@ -53,28 +59,25 @@
   public static final String CUSTOM_JAVA_FILE_BASED_SINK_URN =
       "urn:beam:file_based_sink:javasdk:0.1";
 
-  public static final String CUSTOM_JAVA_FILE_BASED_SINK_FORMAT_FUNCTION_URN =
-      "urn:beam:file_based_sink_format_function:javasdk:0.1";
-
   @VisibleForTesting
   static WriteFilesPayload toProto(WriteFiles<?, ?, ?> transform) {
+    Map<String, SideInput> sideInputs = Maps.newHashMap();
+    for (PCollectionView<?> view : transform.getSink().getDynamicDestinations().getSideInputs()) {
+      sideInputs.put(view.getTagInternal().getId(), ParDoTranslation.toProto(view));
```

Comment:
<!--thread_id:cc_128886805_t; commit:3708e20aa6a660b2f1e3519656675abc8887ab17; resolved:0-->
<!--section:context-quote-->
> **reuvenlax** wrote:
> Ok, so I should move this method into PCollectionViewTranslation, but some caveats should be added to the method comment? Which caveats?

<!--section:body-->
I think I misread something here. This method seems fine as-is once in PCollectionViewTranslation.